Jenkinsfile

```diff
@@ -200,9 +200,9 @@ pipeline {
                                     script {
                                         // If this build is not a pull request, run sonar scan. otherwise run incremental scan
                                         if (env.CHANGE_ID == null) {
-                                            sh 'mvn -q -B -Dfindbugs.skip=true -Dcheckstyle.skip=true org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar -Dsonar.host.url=https://sonarqube.com -Dsonar.login=$SONAR_TOKEN  -Dsonar.organization=codice -Dsonar.projectKey=ddf -pl !$DOCS,!$ITESTS'
+                                            sh 'mvn -q -B -Dfindbugs.skip=true -Dcheckstyle.skip=true org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar -Dsonar.host.url=https://sonarqube.com -Dsonar.login=$SONAR_TOKEN  -Dsonar.organization=codice -Dsonar.projectKey=ddf -Dsonar.coverage.exclusions=**/test/**,**/itests/**,**/*Test*,**/sdk/**,**/*.js,**/node_modules/**,**/jaxb/,**/wsdl/,**/nces/sws/**,**/*.adoc,**/*.txt,**/*.xml -pl !$DOCS,!$ITESTS'
```

Comment:
Could we put that list of exclusions in a var up at the top and then just use that here and below?

Reply:
I thought I could get one by you, but you foiled me again :shakes_fist:

Will update